tests/models/flow_cell/test_flowcell_model.py

```diff
@@ -146,6 +149,30 @@ def test_get_run_parameters_when_non_existing(demultiplexed_runs: Path):
         flow_cell.run_parameters
 
 
+@pytest.mark.parametrize(
+    "flow_cell_fixture, expected_sequencer",
+    [
+        ("hiseq_2500_flow_cell", Sequencers.HISEQGA),
+        ("hiseq_x_flow_cell", Sequencers.HISEQX),
+        ("novaseq_6000_flow_cell", Sequencers.NOVASEQ),
+        ("novaseq_x_flow_cell", Sequencers.NOVASEQX),
+    ],
+)
+def test_flow_cell_run_parameters_type(
+    flow_cell_fixture: str, expected_sequencer: str, request: FixtureRequest
+):
+    """Test that the run parameters of the flow cell is of teh expected type."""
+    # GIVEN a flow cell without _run_parameters
+    flow_cell: FlowCellDirectoryData = request.getfixturevalue(flow_cell_fixture)
+    assert not flow_cell._run_parameters
```

Comment:
Is the `_run_parameters` intentional?

Reply:
Not sure either, but I guess we wanted to separate the attribute from the getter. I will leave it for now and see if it is worth refactoring later